### PR TITLE
feat(web): button to delete all events in plan

### LINKED ISSUE #94: Users should be able to clear all their events
### Expected Behavior

Clear button for all events next to plan title

image

### Current Behavior

no clear button, users need to clear each one individually

### PR DESCRIPTION
**Pull request by pbebla**
### Description

A button in the top right of the plan card to delete all events instead of deleting each event individually.

### Tested

After adding a couple of events, clicking the new button returns to the default empty list.

### Related issues

* Fixes #94

### CODE REVIEW

File: packages/web/src/components/Plan.tsx [hunk 1]
```diff
@@ -189,6 +190,10 @@ export class Plan extends React.Component<{}, State> {
         this.setState({ events: removeEvent(this.state.events, id) });
     };
 
+    removeAllEvents = (): void => {
+        this.setState({ events: removeAllEvents(this.state.events) });
```
Comment (drklee3) [inline packages/web/src/components/Plan.tsx]: Does this work if you just set events to an empty array?

```js
this.setState({ events: [] });
```

So this way you won't need to add / use an extra `removeAllEvents` function

**Review by drklee3 — COMMENTED** [review]
Looks good! Just 1 small comment

File: packages/web/src/components/Plan.tsx (hunk 1 above)
Comment (pbebla) [reply packages/web/src/components/Plan.tsx]: yeah that works

**Review by drklee3 — APPROVED** [review]
lgtm!

**Comment by drklee3** [comment]
@sportsman2325 I think you'll need to resolve the merge conflict before it can be merged in, should be just the imports.

**Pull request merged**